onnxruntime/python/tools/tensorrt/perf/ort_build_latest.py

```diff
@@ -1,17 +1,36 @@
 import os
 import subprocess
 import argparse
+from perf_utils import get_latest_commit_hash
 
 def parse_arguments():
     parser = argparse.ArgumentParser()
 
     parser.add_argument("-o", "--ort_master_path", required=True, help="ORT master repo")
     parser.add_argument("-t", "--tensorrt_home", required=True, help="TensorRT home directory")
     parser.add_argument("-c", "--cuda_home", required=True, help="CUDA home directory")
-
+    parser.add_argument("-v", "--commit_hash", required = False, help="Github commit to test perf off of")
+    parser.add_argument("-s", "--save", required=False, help="Directory to archive wheel file")
+    parser.add_argument("-a", "--use_archived", required=False, help="Archived wheel file")
     args = parser.parse_args()
     return args
 
+def archive_wheel_file(save_path, ort_wheel_file):
+    if not os.path.exists(save_path):
+        os.mkdir(save_path)
+    p1 = subprocess.Popen(["cp", ort_wheel_file, save_path])
```

Comment:
general comment, consider using the higher-level function [subprocess.run()](https://docs.python.org/3/library/subprocess.html#subprocess.run) and verifying that it succeeded (e.g., `subprocess.run(..., check=True)`)
for "cp", consider one of the functions in [shutil](https://docs.python.org/3/library/shutil.html)

Reply:
other than cp, 
please make Ed's suggestion to use subprocess.run() rather than popen() and then wait()
for all places in the code.